setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = magql
-version = 0.4.0
+version = 0.5.0
```

Comment:
0.4.1?

Reply:
I pushed to pypi with 0.4.1, but I thought we were going with the version that was one ahead of what we were at now. Like how we had 0.4.0 while we working on developing it.

Reply:
Ah, I thought this was the 0.4.1 release because you were adding the changelog.

Reply:
It is supposed to be the 0.4.1 release. I didn't bump the version to 0.5.0 in the 0.4.0 release but should have. This is making up for that. I think this is all correct.